test/logging.test.js

```diff
@@ -128,15 +127,13 @@ describe('logging', () => {
 							expect(client._uniqueLogs).to.be.empty;
 							done();
 						}
-					};
-					const client = new LoggingClient('my-app-id', mockLogger);
+					});
```

Comment:
The `mockLogger` was using `client` before it was defined. So moving it down into the same line fixes that.